lib/fake_braintree/subscription.rb

```diff
@@ -95,7 +95,7 @@ def discounts_or_add_ons(discount_or_add_on)
             'amount'   => hsh['amount']
           }
         end
-      elsif discount_or_add_on['update']
+      elsif Array(discount_or_add_on['update']).any?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.